src/coach/defence/DefenceDealer.cpp

```diff
@@ -63,7 +63,7 @@ void DefenceDealer::updateDefenderLocations() {
     auto foundDefenders=g_defensivePositionCoach.decidePositions(lockedDefenders, freeDefenders);
     availableIDs.clear();
     if (foundDefenders.size()<freeDefenders.size()+lockedDefenders.size()){
-        std::cout<<"WTFF"<<std::endl;
+        std::cout<<"[DefenceDealer] Warning: There is no opponent to defend against"<<std::endl;
```

Comment:
This should probably be something like 'DefenceDealer could not find a valid position on the field for one or more robots", again not hugely important